features/utils/graphql_client.py

```diff
@@ -0,0 +1,36 @@
+from six.moves import urllib
+import json
+
+
+class GraphQLClient:
```

Comment:
Why handcraft a graphql client from scratch instead of using something already done : https://github.com/prisma-labs/python-graphql-client

Reply:
if you look at the code in this repo, you will notice that our code is a copy / paste of that code; I don't want to have a submodule containing just that; in the end it's just a simple graphql client, nothing more.